contrib/debian/README.md

```diff
@@ -15,7 +15,7 @@ To install:
 
 If you build yourself, you will either need to modify the paths in
 the .desktop file or copy or symlink your dash-qt binary to `/usr/bin`
-and the `../../share/pixmaps/dash128.png` to `/usr/share/pixmaps`
+and the `../../share/pixmaps/bitcoin128.png` to `/usr/share/pixmaps`
```

Comment:
yep, will require to fix winfows packaging though because of bitcoin.ico (Makefile.am and setup.nsi.in)... or let's start with bitcoin*.png and bitcoin*.xpm for debian packaging only changes :)